packages/mediapipe-core/analysis_options.yaml

```diff
@@ -0,0 +1,24 @@
+# This file configures the static analysis results for your project (errors,
+# warnings, and lints).
+#
+# This enables the 'recommended' set of lints from `package:lints`.
+# This set helps identify many issues that may lead to problems when running
+# or consuming Dart code, and enforces writing Dart using a single, idiomatic
+# style and format.
+#
+# If you want a smaller set of lints you can change this to specify
+# 'package:lints/core.yaml'. These are just the most critical lints
+# (the recommended set includes the core lints).
+# The core lints are also what is used by pub.dev for scoring packages.
+
+include: package:lints/recommended.yaml
+
+# Uncomment the following section to specify additional rules.
+
+# linter:
+#   rules:
+#     - camel_case_types
+
+analyzer:
+  exclude:
+    - "lib/src/mediapipe_common_bindings.dart"
```

Comment:
Why for do we exclude this?

Reply:
It's generated code, so occasionally it will behind on our latest best practices in `flutter_lint` and cause IDE analysis engine noise.